valuesets/ValueSet-England-ODSRoleCode.xml

```diff
@@ -0,0 +1,29 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+	<id value="England-ODSRoleCode" />
+	<url value="https://fhir.nhs.uk/England/ValueSet/England-ODSRoleCode" />
+	<version value="1.0.0" />
+	<name value="EnglandODSRoleCode" />
+	<title value="England ODS Role Code" />
+	<status value="draft" />
+	<experimental value="false" />
+	<date value="2023-10-13" />
+	<publisher value="NHS England" />
+	<contact>
+		<name value="NHS England" />
+		<telecom>
+			<system value="email" />
+			<value value="interoperabilityteam@nhs.net" />
+			<use value="work" />
+			<rank value="1" />
+		</telecom>
+	</contact>
+	<description value="A valueset to identify the Organisation Data Service (ODS) role codes" />
+	<purpose value="A valueset to identify the Organisation Data Service (ODS) role codes" />
+	<copyright value="Copyright © 2023+ NHS England Licensed under the Apache License, Version 2.0 (the \&quot;License\&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \&quot;AS IS\&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
+	<compose>
+		<include>
+			<system value="https://fhir.nhs.uk/England/CodeSystem/England-ODSRoleCode" />
+			<version value="1.0.0" />
```

Comment:
Don't believe version line is needed because not needed as part of the same package

Reply:
Will remove the line <version value="1.0.0" />.